### ISSUE TITLE
project bugs

### ISSUE THREAD

**Issue by yzbx** [ISCAS007/torchseg#25]
# torchseg
- [ ] data augmentation ?
  - data augmentation not improve miou abviously!
- [ ] summary image ?
  - for new version of pytorch, summary image cause error!
  - currently forbidden image summary
- [ ] keras version pspnet
  - may run slow and get low miou
  - bugs in metric recall, currently use numpy metric function instead

**Comment by yzbx** [comment]
# deeplab
- [ ] metric summary when mode=train for pspnet.py
  - summary failed for train, use evaluation instead, but hooks work correctly!
